backend/de.metas.externalsystem/src/main/sql/postgresql/ddl/views/ExternalSystem_Config_PInstance_Log_v.sql

```diff
@@ -12,15 +15,48 @@ SELECT pi.ad_process_id,
        pi.ad_pinstance_id,
        pi.isprocessing,
        pi.errormsg,
-       now() as created,
-       100 as createdby,
-       now() as updated,
-       100 as updatedby,
-       'Y' as isactive
-FROM ad_pinstance pi
+       i.ad_issue_id                                         as ad_issue_id,
+       i.issuesummary                                        as issuesummary,
+       'Error'                                               as type,
+       now()                                                 as created,
+       100                                                   as createdby,
+       now()                                                 as updated,
+       100                                                   as updatedby,
+       'Y'                                                   as isactive
+from ad_pinstance pi
          JOIN ad_pinstance_para pip_request ON pi.ad_pinstance_id = pip_request.ad_pinstance_id AND pip_request.parametername = 'External_Request'
+         JOIN ad_issue i on i.ad_pinstance_id = pi.ad_pinstance_id
          LEFT JOIN ad_pinstance_para pip_config ON pi.ad_pinstance_id = pip_config.ad_pinstance_id AND pip_config.parametername = 'ChildConfigId'
```

Comment:
as AD_PIstance.Record_ID and AD_Table_ID are always set, i think we also don'tneed to join this anymore..right?